modin/pandas/test/test_io.py

```diff
@@ -818,6 +818,24 @@ def test_read_csv_iteration(self, iterator):
 
         df_equals(modin_df, pd_df)
 
+    def test_read_csv_encoding_976(self):
```

Comment:
I have tried to run this test both on Windows and Linux on the current master and it passed each time. Is it only MacOS issue?

Reply:
It's strange. I reproduced this issue on Windows and Linux.

Reply:
How are you create `modin/pandas/test/data/issue_976.csv` file for tries?

Reply:
If you create a file copying the content, then the tests pass on the master as well. Try to download the file from issue.

Reply:
Yes, you are right, copying is not correct, with downloaded file test works fine.